modules/tinymce/src/plugins/lists/main/ts/core/Delete.ts

```diff
@@ -273,7 +273,18 @@ const hasListSelection = (editor: Editor) => {
 const backspaceDeleteRange = (editor: Editor): boolean => {
   if (hasListSelection(editor)) {
     editor.undoManager.transact(() => {
+      // Some delete actions may prevent the input event from being fired. If we do not detect it, we fire it ourselves.
+      let fireInput = true;
+      const inputHandler = () => fireInput = false;
+
+      editor.on('input', inputHandler);
```

Comment:
Since the `delete` command is our own command here is that we missed firing input on some override we make? So it fires when it's native only or something.

Reply:
Right we made that input patch only for keyboard commands. Since this is also a keyboard action but not in core we don't fire one. I guess some browser do and some others dont?

Reply:
We default to the browser's native delete action in the delete command, where we very explicitly block the `beforeInput` and `input` events ( See `tinymce/modules/tinymce/src/core/main/ts/delete/DeleteUtils.ts:13` at `execCommandIgnoreInputEvents` which we get to from `execNativeDeleteCommand` )